include/roboteam_utils/Arc.h

```diff
@@ -13,7 +13,7 @@ namespace rtt {
  * This is because, appearantly, the fmodl function only has float precision even though it should support
  * 128-bit long doubles...
  */
-    constexpr double ARC_MAX = M_PI - std::numeric_limits<float>::epsilon();
+    constexpr double ARC_MAX = 2*M_PI - std::numeric_limits<float>::epsilon();
```

Comment:
what does this change mean?

Reply:
This is one I'm not entierly sure about. Tests were failing, but also the range of this class in the calculations was assumed to be 0 to 2 PI, so I think this is the correct solution.

Reply:
I looked into it a bit further and yeah... this class needs a lot more help. I'm pretty sure this is correct as in normalize() the angle is clamped between 0 and 2M_PI and not between 0 and PI.

Reply:
Ok that's good. Still, someone needs to fix this class as there are unfinished functions etc.